packages/product-react/src/Tabs/TabView.test.tsx

```diff
@@ -0,0 +1,77 @@
+import Enzyme, { configure } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+import React from 'react';
+import { TabContext } from './context';
+import { TabView } from './TabView';
+
+configure({ adapter: new Adapter() });
+
+interface IContextProviderProps {
```

Comment:
nit: prefer `type` over `interface`

Reply:
https://medium.com/@martin_hotell/10-typescript-pro-tips-patterns-with-or-without-react-5799488d6680 #19 has some really good points

Reply:
If we want to enforce this we should set up linting for it. Interface has been the paradigm i've seen us use so far